README.md

```diff
@@ -1 +1,23 @@
 # TL Projects UX Work
+Two ways to open Sketch file in your local laptop: 
```

Comment:
@victoriali8023 Let's add the Readme from the top-level to ccm Readme. The info provided here is generic but I am not sure yet how the repo evolves for UX work and what tools they prefer.

Reply:
Okay, I will revise the my Readme. As for the future tool, Adobe XD and Sketch may still be two main options for designers (if Figma is not secure).